src/main/java/auxiliary/Figure.java

```diff
@@ -0,0 +1,19 @@
+package auxiliary;
+
+public abstract class Figure implements Draw, GetArea {
+    protected String color;
```

Comment:
let's make this field private and create a constructor in this class for color field